### ISSUE TITLE
UX to expand out API endpoint docs

### ISSUE THREAD

**Issue by harry-wood** [transportapi/api-console#6]
To see actual endpoint docs you have to click to expand out the section.  e.g. you see "/uk/bus/stop/{atcocode}/live.json" and then have to click on the little tab on the right hand side which says "GET".   Even to me, having used to docs a lot, I find that UX counter-intuitive. Why can't I click on the "/uk/bus/stop/{atcocode}/live.json" title?

....and given that all our endpoints are GET, could we actually remove those GET tabs? For a noddy API user, talk of GET is unnecessarily confusing and cluttering.

Could be an idea to suggest upstream

**Comment by madwort** [comment]
I agree that ideally the title would be clickable, but I don't think it's a massive problem - looking at those docs you expect something to be clickable so just have a little search. Not ideal but not blocking for me. Feature request upstream? 

I think it needs to say `GET`.

**Comment by mgogov** [comment]
I see the point on clickability IMO, it's not that painful to have those `GET`s.

It also kindof makes it a bit more clear that what we support is HTTP GET and not some other HTTP verb.

Given the importance of other API console issues, I'd say this is very low priority.
